content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+You can check the health of any Collector instance by checking its own internal
+telemetry. Read on to learn how to use this data to diagnose and troubleshoot
+Collector issues.
+
+## Activate internal telemetry in the Collector
+
+By default, the Collector exposes its own telemetry in two ways:
+
+- Internal [metrics](#configure-internal-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configure-internal-logs) are emitted to `stdout` by default.
+
+[Traces](#configure-internal-traces) are not exposed by default. Two feature
+gates offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
+
+### Configure internal metrics
+
+Prometheus metrics are exposed locally on port `8888` and path `/metrics`. For
+containerized environments, you might want to expose this port on a public
+interface instead of only locally.
```

Comment:
Is that clear enough? For example, `127.0.0.1` and `0.0.0.0` could both be interpreted as "locally", however the former causes issues in some cases in the context of containerized environments.

Reply:
How about the following, which I have copied from [another section](https://opentelemetry.io/docs/collector/configuration/#telemetry) of the docs website? (@theletterf @svrnm Do we care about redundancy here?)

> You can configure how internal metrics are generated and exposed by the Collector. By default, the Collector generates basic metrics about itself and exposes them for scraping at http://127.0.0.1:8888/metrics. You can expose the endpoint to one specific or all network interfaces when needed. For containerized environments, you might want to expose this port on a public interface. 

I can then add some additional color to the sample config that follows.

Reply:
Thanks, works for me.